deploy/cr-minimal.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: minimal-cluster
 spec:
   crVersion: 1.12.0
-  image: percona/percona-server-mongodb:4.4.10-11
+  image: percona/percona-server-mongodb:5.0.7-6
   allowUnsafeConfigurations: true
   upgradeOptions:
     apply: 4.4-recommended
```

Comment:
The PR should be from your branch to release branch.
Please change the `4.4-recommended` to `5.0-recommended` (same in cr.yaml)